contact.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable linebreak-style */
+const contact = document.querySelector('.contact');
```

Comment:
- [ ] You have disabled too many lines of code while you were trying to fix eslint errors, kindly try to fix eslint errors without disabling many lines of code. 

- check the `index.js and contact.js file`